modules/microshift-rhoai-verify-model-connected.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhoai-verify-model-connected_{context}"]
+= Verifying that your AI model is accessible
+
+Before querying the model through the API, you can check to be certain that the model is accessible and ready to provide answers based on the connected data. The following examples continue with the {ovms}.
+
+.Prerequisites
+
+* You configured the AI model-serving runtime.
+* You uploaded your AI model to {microshift-short}.
+* The {microshift-short} cluster is running.
+* You installed {oc}.
+* You installed `jq`.
+
+.Procedure
+
+. Get the IP address of the {microshift-short} cluster and assign it to the `IP` variable as the following example shows:
++
+[source,terminal]
+----
+$ IP=$(oc get nodes -o json | jq -r '.items[0].status.addresses[0].address')
+----
+
+. Identify the name of the route you created by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get route _<route_name>_ -o yaml <1>
+----
+<1> Replace <route_name> with the actual name of your route.
+
+. Extract the `Host` value of the route by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get route _<route_name>_ -o yaml | jq '.spec.host' <1>
+----
+<1> Replace _<route_name>_ with the actual name of your route.
+
+. Assign the `HOST` value of the route to the `DOMAIN` variable by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ DOMAIN=$(oc get route _<route_name>_ -o yaml | jq '.spec.host') <1>
+----
+<1> Replace _<route_name>_ with the actual name of your route.
+
+. Enable data transfer from the route to the {microshift-short} IP address by running the following command:
++
+----
+DOMAIN=ovms-resnet50-predictor-ai-demo.apps.example.com \
+IP=192.168.0.10 \
+curl -i "${DOMAIN}/v2/models/ovms-resnet50/ready" \
+    --connect-to "${DOMAIN}::${IP}:" \ //<1>
```

Comment:
In previous steps we assigned IP and DOMAIN variables, we shouldn't overwrite them here. Also trailing `\` needs to be removed as shell would expect another line in the command

```suggestion
curl -i "${DOMAIN}/v2/models/ovms-resnet50/ready" \
    --connect-to "${DOMAIN}::${IP}:" //<1>
```